packages/amplication-client/src/VersionControl/PendingChanges.scss

```diff
@@ -77,8 +77,9 @@ $resource-name-max-width: 180px;
     justify-content: flex-start;
     align-items: center;
 
-    img {
-      height: $empty-state-image-size;
+    .svg-theme-image {
+      width: 120px;
```

Comment:
you have a variable at the top of this file that its value is 120px
`$empty-state-image-size: 120px;`
please use it.

Why did you add the width?

Reply:
My bad, fixed the issue and used the variable